### PR TITLE
Test if try register ex bucket with non exist listing throw user err

### PR DESCRIPTION
**Pull request by romanbracinik**
Jira: BIG-197
KBC: https://github.com/keboola/connection/pull/4708

Before asking for review make sure that:

## Checklist

- [x] New client method(s) has tests
- [x] Apiary file is updated

## Release

  - [x] I gave the PR a proper label:
    * major (BC break)
    * minor (new feature)
    * patch (backwards compatible fix)
    * no release (just test changes)

### CODE REVIEW

File: tests/Backend/ExternalBuckets/BigqueryRegisterBucketTest.php [hunk 1]
```diff
@@ -118,6 +118,46 @@ public function provideComponentsClientTypeBasedOnSuite(): array
         return (new TestSetupHelper())->provideComponentsClientTypeBasedOnSuite($this);
     }
 
+    /**
+     * @dataProvider provideComponentsClientTypeBasedOnSuite
+     */
+    public function testIfTryRegisterNonExistingListingItFailWithUserErr(string $devBranchType, string $userRole): void
```
Comment (jirkasemmler) [inline tests/Backend/ExternalBuckets/BigqueryRegisterBucketTest.php:124]:
```suggestion
    public function testRegisterNonExistingListingFailsWithUserError(string $devBranchType, string $userRole): void
```
```suggestion
    public function testRegisterNonExistingListingFailsWithUserErr(string $devBranchType, string $userRole): void
```

Comment (jirkasemmler) [reply tests/Backend/ExternalBuckets/BigqueryRegisterBucketTest.php:124]: jen at ten nazev neni tak dlouhy a dava vetsi smysl.

Comment (romanbracinik) [reply tests/Backend/ExternalBuckets/BigqueryRegisterBucketTest.php]: Jo jasne dik, takto je to lepsie :)

**Review by jirkasemmler — APPROVED** [review]
jen dve poznamky
- ten naming convention, at se tam vyvarovavame zbytecny vaty
- prosim drzme konvence : a) ID jira issue v nazvy PR b) ID jira issue jako prefix branche

**Pull request merged**